cypress/integration/seededFlows/loggedOutFlows/namespacedForemMobile.spec.js

```diff
@@ -0,0 +1,109 @@
+describe('Namespaced ForemMobile functions', () => {
```

Comment:
Tests are all green now. 😎  As mentioned, in Zoom, merge latest because to get rid of the application error we were seeing while running some of the tests.

Also, as mentioned, I'm not sure why, but some of the tests require you to be logged in, which I think you weren't expecting them to be, i.e. `win.ForemMobile.injectJSMessage({ action: 'test' })`

Also, as discussed, since this includes tests where the user is logged in, it doesn't make much sense being in the `loggedOutFlows` folder. Consider another folder like `mobileBridging` or whatever you think makes sense.

![image](https://user-images.githubusercontent.com/833231/142931422-89a5e21b-e296-4a10-be0f-73494512ddd2.png)